cpp/ps0001/p301F solving problem 1.cpp

```diff
@@ -4,21 +4,22 @@ using namespace std;
 
 int main()
 {
-    int num1;
-    int num2;
-    int num3;
-
-
-cout<<"enter the first number /n1" << endl;
-cin>> num1;
-
-cout<<"enter the second number /n1" << endl;
-cin>>num2;
-
-
-cout<<"enter the third number /n1" << endl;
-cin>> num3;
-
-    cout << num1+num2+num3 << endl;
+    // prompting users for 3 numbers to add 1/2
+    short num1;
+    short num2;
+    short num3;
+    
+    cout << "enter the first number" << endl;
+    cin >> num1;
+    
+    cout << "enter the second number" << endl;
+    cin >> num2;
+    
+    cout << "enter the third number" << endl;
+    cin >> num3;
+    
+    // outputting the result of the summation process 3/3
+    cout << num1 + num2 + num3 << endl;
     return 0;
 }
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [ ] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code